python/publish/publisher.py

```diff
@@ -305,6 +313,14 @@ def publish_json(self, data: PublishData):
         # provide a reduced version to Github actions
         self._gha.set_output('json', json.dumps(data.reduced(), ensure_ascii=False))
 
+    def publish_summary(self, title: str, summary: str):
+        if self._settings.job_summary_file:
+            try:
+                with open(self._settings.job_summary_file, 'a+', encoding='utf-8') as summary_file:
+                    summary_file.write( f'## {title}\n{summary}\n')
```

Comment:
```suggestion
                    summary_file.write(f'## {title}\n{summary}\n')
```

Reply:
In the end, just putting the summary is more than enough :)